frontend/src/components/MainSearch/MainSearch.js

```diff
@@ -67,11 +67,7 @@ class MainSearch extends Component {
   renderSearchInstructions() {
     return (
       <div className="searchInstructions">
-        <p>
-          You can search recipe names or ingredients by keyword. Enter your keywords and click the
-          search button or press enter to search. If you use multiple keywords, the search will
-          return only results containing all of your keywords.
-        </p>
+        <p> You can search recipe names or ingredients by keyword. Enter your keywords and click the search button or press enter to search. If you use multiple keywords, the search will return only results containing all of your keywords.</p>
```

Comment:
Maybe your editor is changing these automatically? If so, I'd turn off that feature, and keep this as a multi-line paragraph.